### PR TITLE
NCDA for CHW - vaccination history activity

### PR DESCRIPTION
**Pull request by anvmn**
#780

### CODE REVIEW

**Comment by anvmn** [comment]
@AronNovak This PR is big because it's about single activity that got 7 tasks of same type (differrent vaccinations), so it was not wort to break it into 7 PRs.
There're also some code relocations (to allow multiple modules to use same utility functions), so many files are different only  at the `import` section.

File: client/src/elm/Pages/ChildScoreboard/Activity/Model.elm [hunk 1]
```diff
@@ -2,29 +2,91 @@ module Pages.ChildScoreboard.Activity.Model exposing (..)
...
+    , opvForm = emptyVaccinationForm
+    , pcv13Form = emptyVaccinationForm
+    , rotarixForm = emptyVaccinationForm
+    , activeTask = Nothing
     }
 
 
 type Msg
-    = SetActivePage Page
+    = NoOp
```
Comment (AronNovak) [inline client/src/elm/Pages/ChildScoreboard/Activity/Model.elm:66]: Where do we use this `NoOp`? It seems to be an outlier compared to the other messages. Do we really need it?

Comment (AronNovak) [reply client/src/elm/Pages/ChildScoreboard/Activity/Model.elm:66]: Ah, I see, is it temporary then?

**Review by AronNovak — APPROVED** [review]

File: client/src/elm/Pages/ChildScoreboard/Activity/Model.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/ChildScoreboard/Activity/Model.elm:66]: @AronNovak 

Well, no, it's not temporary. This is 'empty' message - similar to concept of `emptyNode`.
There are rare cases where we need the `NoOp` (no operation), as some of logical pathes should result in action while others are not. For example:

Selection_2100

**Pull request merged**